index.js

```diff
@@ -15,8 +22,22 @@ client.on('message', (message) => {
     if (!message.guild) return;
     if (!message.content.startsWith(prefix)) return;
     const msg = message.content.substr(prefix.length);
+    const [_, ...args] = msg.split(" ");
+    const command = commands.get(msg.split(" ")[0]);
```

Comment:
I believe this and [this](https://github.com/yavrib/discord-music-streaming-bot/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR25) are trying to achieve the same thing